src/styles.ts

```diff
@@ -15,6 +23,16 @@ export const headlineFontStyles = css`
     margin: 0;
 `;
 
+export const textPadding = css`
+    padding-left: ${spaceToRem(2)};
+    padding-right: ${spaceToRem(2)};
```

Comment:
how much is one space? we've got `basePx` of 8px throughout the codebase.

Reply:
A unitless `4`

Reply:
great, we should:
```ts const basePx = spaceToRem * 2;```